packages/typescript-config/nextjs.json

```diff
@@ -8,6 +8,7 @@
     "moduleResolution": "Bundler",
     "allowJs": true,
     "jsx": "preserve",
-    "noEmit": true
+    "noEmit": true,
+    "verbatimModuleSyntax": true
```

Comment:
Forces using `import type` and `export type`, leading to more predictable behavior and fewer unnecessary imports.